test/SlowTests/Server/Documents/Revisions/RevisionsReplication.cs

```diff
@@ -266,6 +266,13 @@ public async Task ConflictedRevisionClearedAfterEnforce()
             using (var store1 = GetDocumentStore())
             using (var store2 = GetDocumentStore())
             {
+                var config2 = new RevisionsCollectionConfiguration
+                {
+                    Disabled = false,
+                    MinimumRevisionsToKeep = 0
+                };
+                await RevisionsHelper.SetupConflictedRevisions(store1, Server.ServerStore, configuration: config2);
+
```

Comment:
The original test wasn't failing because the revisions deletion behavior was wrong

Reply:
in what sense?

Reply:
In the original test, After 2 ways external replication you get a doc with conflict/resolved revisions in the 2 databases.
After the `enforce-configuration` you expect the revisions will be deleted - that assumption is wrong because the `conflict revisions config` you have (save all the conflicted/resolved revisions from the last 45 days) shouldn't delete the revisions.
Only if you change the config before the `enforce-configuration` to configuration like `config2`, the revisions should be deleted.

**Another comment about this test:**
Although we change the `conflicted-revisions` config before the conflicted/resolved revisions are created, the revisions aren't removed immediately because the config isn't applied on replication, only on `put/delete/Enforce-Configuration`